data/reference/syntax.adoc

```diff
@@ -0,0 +1,270 @@
+= AsciiDoc Syntax
+
+== Paragraphs
+
+A normal paragraph.
+
+  An indented literal
+  paragraph.
+
+== Formatted Text
+
+*bold text*.
+_emphasized text_.
+`monospaced text`.
+
+=== Unconstrained
+
+In short, unconstrained means anywhere, including within a word:
+
+bold ** text with spaces **.
+emphasized le__tt__ers.
+m``o``n``o``spaced letters.
+
+== Document Links
+
+// define link target
+[[id]]
+// link to target id
+<<id,caption>>
+// link to an external file
+link:file#id[caption]
+
+=== URLs
+
+Use normal URL and email address syntax or:
+
+mailto:hello@domain.org[Say hello!]
+link:https://domain.org/page[Go to page]
+link:../path/to/file.txt[Read file]
+link:\\path\to\file.pdf[Download file.pdf]
```

Comment:
I think we should cover the interdoc xref here with an example like `xref:sibling-document.adoc[Sibling Document]` or something to that affect.

Reply:
done